make.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+#source env.properties
+PROJECT="JWT"
+VER="1.0.0"
+
+
+CP="classes:"
+jars=`ls lib/*.jar`
+for i in $jars; 
+do  
+	CP="$CP:$i"
+done
+
+#clear
+echo " "
+echo "Project : $PROJECT performing task [ $1 ] "
+echo " "
+echo "Classpath : $CP "
+
+function usage() {
+	echo "Usage : ./make.sh [target]"
+	echo "where target : one of [ clean compile ]"
+}
+
+function clean() {
+	rm -rf classes/com/*
```

Comment:
@marcotroisi - Laziness on my part. I will make a jar of the classes needed and remove this from the repo